cypress/e2e/functional/tile_tests/table_tool_spec.js

```diff
@@ -125,6 +125,21 @@ context('Table Tool Tile', function () {
     tableToolTile.getTableCell().eq(1).should('contain', 'third value');
     tableToolTile.getTableCell().eq(1).should('not.contain', 'abandon this edit');
 
+    cy.log('can press enter key for edit mode without adding newline or otherwise altering text');
```

Comment:
would it be worth adding a test for the improved 'tab' key handling too?

Reply:
FYI: [Press tab in your tests: Introducing cy.press()](https://www.cypress.io/blog/press-tab-in-your-tests-introducing-cy-press).